taskcluster/client.py

```diff
@@ -69,6 +76,11 @@ def __init__(self, options=None):
             log.debug('credentials key scrubbed from logging output')
         log.debug(dict((k, v) for k, v in o.items() if k != 'credentials'))
 
+        if session:
+            self.session = options.session
```

Comment:
yes.  that was a typo from addressing review feedback.  this would still work if you pass in the session as part of options, but thats not the desired situation.